packages/sku/src/program/commands/start-ssr/webpack-start-ssr-handler.ts

```diff
@@ -178,6 +177,14 @@ export const webpackStartSsrHandler = async ({
     setupExitSignals: true,
   };
 
+  devServerConfig.setupMiddlewares = (middlewares) => [
+    ...middlewares,
+    {
+      name: 'send-to-ssr',
+      middleware: proxy(serverHost),
+    },
+  ];
+
```

Comment:
Likely leftover from before making the config always-on, but can this be moved into the `devServerConfig` definition instead of mutating it?